### PR TITLE
Add Comelit alarm information

### PR DESCRIPTION
**Pull request by chemelli74**
## Proposed change

Add Comelit alarm information

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/104718
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/comelit.markdown [hunk 1]
```diff
@@ -23,5 +24,10 @@ The Comelit SimpleHome integration allows you to control your [Comelit home auto
 There is support for the following platform types within Home Assistant:
 
 - **Comelit Serial Bridge** - allows local control for light, cover, irrigation and "other" devices; creates power sensors.
+- **Comelit VEDO System** - allows local control of alarm system
 
 {% include integrations/config_flow.md %}
+
+## Alarm control panel ##
+
+The integration will create an alarm entity for each "area" and a sensor for each "zone".
```
Comment (c0ffeeca7) [inline source/_integrations/comelit.markdown]:
```suggestion
The integration will create an alarm entity for each {% term area %} and a sensor for each {% term zone %}.
```
We have thsre term refs now, pointing to the glossary :book:

File: source/_integrations/comelit.markdown [hunk 2]
```diff
@@ -23,5 +24,10 @@ The Comelit SimpleHome integration allows you to control your [Comelit home auto
 There is support for the following platform types within Home Assistant:
 
 - **Comelit Serial Bridge** - allows local control for light, cover, irrigation and "other" devices; creates power sensors.
+- **Comelit VEDO System** - allows local control of alarm system
```
Comment (c0ffeeca7) [inline source/_integrations/comelit.markdown]:
```suggestion
- **Comelit VEDO System** - allows local control of the alarm system.
```

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @chemelli74 :+1:

File: source/_integrations/comelit.markdown [hunk 3]
```diff
@@ -23,5 +24,10 @@ The Comelit SimpleHome integration allows you to control your [Comelit home auto
 There is support for the following platform types within Home Assistant:
 
 - **Comelit Serial Bridge** - allows local control for light, cover, irrigation and "other" devices; creates power sensors.
+- **Comelit VEDO System** - allows local control of the alarm system.
 
 {% include integrations/config_flow.md %}
+
+## Alarm control panel ##
```
Comment (c0ffeeca7) [inline source/_integrations/comelit.markdown]:
```suggestion
## Alarm control panel
```

File: source/_integrations/comelit.markdown [hunk 4]
```diff
@@ -23,5 +24,10 @@ The Comelit SimpleHome integration allows you to control your [Comelit home auto
 There is support for the following platform types within Home Assistant:
 
 - **Comelit Serial Bridge** - allows local control for light, cover, irrigation and "other" devices; creates power sensors.
+- **Comelit VEDO System** - allows local control of the alarm system.
 
 {% include integrations/config_flow.md %}
+
+## Alarm control panel
+
+The integration will create an alarm entity for each {% term area %} and a sensor for each {% term zone %}.
```
Comment (frenck) [inline source/_integrations/comelit.markdown]: Adjusted this one, as these are zones from the alarm system, not Home Assistant (which the terms refer to).

```suggestion
## Alarm control panel

The integration will create an alarm entity for each area and a sensor for each zone.
```

**Review by frenck — APPROVED** [review]

**Pull request merged**